### PR TITLE
Add pr and ci support for Arm64 Azure VM runs

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

**Comment by DrewScoggins** [comment]
@sebastienros PTAL

**Comment by DrewScoggins** [comment]
This still needs to wait for the service connection to be added to the AzDO by the engineering team.

File: build/azure.profile.yml [hunk 1]
```diff
@@ -50,3 +50,53 @@ profiles:
...
+
+  aspnet-azurearm64-lin-relay:
+    variables:
+      serverPort: 5000
+      serverAddress: 10.0.0.4
+      cores: 4
+    jobs: 
+      db:
+        endpoints: 
+          - https://aspnetperf.servicebus.windows.net/azuredb
```
Comment (sebastienros) [inline build/azure.profile.yml:87]: Did you mean `/azurearm64db` (same comment for the other endpoints)

Comment (DrewScoggins) [reply build/azure.profile.yml:87]: Yes, good catch.

File: build/benchmarks.matrix.azure.yml [hunk 2]
```diff
@@ -15,24 +15,44 @@ groups:
       profiles:
       - aspnet-azure-lin
 
+    - name: Trends Database Azure Arm64 Linux
```
Comment (sebastienros) [inline build/benchmarks.matrix.azure.yml:18]: Two jobs per group means it needs two queues at the top of the file. So we need to create another one in AzDo, might need to ask First Responders for that. `ASP.NET Benchmarks Azure Queue 2` for instance (not mentioning a name so we can use it for something else later). And create a new queue in the service bus service for it.

Comment (DrewScoggins) [reply build/benchmarks.matrix.azure.yml:18]: Sure, I will use a generic name for now.

**Pull request merged**